### ISSUE TITLE
Turning on a setting only while scrcpy is running/showing touch input while scrcpy is running

### ISSUE THREAD

**Issue by prdac** [Genymobile/scrcpy#4262]
- [y] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Describe the solution you'd like**
It would be useful to have the "display touch" (not sure for english name because I don't use english in android, but the dev option that shows a circle on the location where you're touching the screen) in scrcpy. Currently you can enable it through adb but there's no way to turn it off automatically when the device is disconnected/scrcpy is closed.
tl:dr I want to display the location of touch input on the screen only when scrcpy is running becuase it helps when you're using the touch input to navigate the device but are looking at the screen using scrcpy.
